src/Devices/WaterSensorData/WaterSensorByGatewayTopbar.js

```diff
@@ -27,11 +27,10 @@ const useStyles = makeStyles((theme) => ({
 const WaterSensorByGatewayTopbar = (props) => {
   const classes = useStyles();
 
-  const bareNodes = props.bareNodes;
-  const coverNodes = props.coverNodes;
-  let activeChip = props.activeChip;
+  let { activeChip, coverNodes, bareNodes, setActiveChip } = props;
 
   useEffect(() => {
```

Comment:
I'm not clear, but this code smells like the old interface where we selected which serial number to show manually, instead of the current interface where the farm code is queried from the onfarm API. Is this just deprecated code?